modules/client-configuration/pages/supported-features-centos.adoc

```diff
@@ -20,51 +20,48 @@ The icons in this table indicate:
-| Remote commands | icon:check[role="green"] | icon:check[role="green"]
-| System package states | ``Salt`` | ``Salt``
-| System custom states | ``Salt`` | ``Salt``
-| Group custom states | ``Salt`` | ``Salt``
-| Organization custom states    | ``Salt`` | ``Salt``
-| System set manager (SSM) | icon:check[role="green"] | icon:check[role="green"]
-| Service pack migration | icon:times[role="danger"] | icon:times[role="danger"]
-| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Traditional`` | icon:check[role="green"]
-| Advanced Virtual Guest Management icon:asterisk[role="none"] | icon:times[role="danger"] | ``Salt``
-| Virtual Guest Installation (Kickstart), as Host OS | ``Traditional`` | ``Traditional``
-| Virtual Guest Installation (image template), as Host OS | ``Traditional`` | icon:check[role="green"]
-| System redeployment ({ay}) | ``Traditional`` | icon:check[role="green"]
+| Feature | {centos}{nbsp}6 | {centos}{nbsp}7 | {centos}{nbsp}8
+| Client | icon:question[role="gray"] (plain CentOS) / icon:check[role="green"] (with {es}) | icon:question[role="gray"] (plain CentOS) / icon:check[role="green"] (with {es}) | icon:question[role="gray"] (plain CentOS) / ``Salt`` (with {es})
+| Operating system packages  |  icon:times[role="danger"] (plain CentOS) / icon:check[role="green"] (with {es}) | icon:times[role="danger"] (plain CentOS)/icon:check[role="green"] (with {es}) | icon:times[role="danger"] (plain CentOS)/``Salt`` (with {es})
+| Registration | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Install packages | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Apply patches | icon:check[role="green"] (third-party service required for errata) | icon:check[role="green"] (third-party service required for errata) | ``Salt`` (third-party service required for errata)
+| Remote commands | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| System package states | ``Salt`` | ``Salt`` | ``Salt``
+| System custom states | ``Salt`` | ``Salt`` | ``Salt``
+| Group custom states | ``Salt`` | ``Salt`` | ``Salt``
+| Organization custom states    | ``Salt`` | ``Salt`` | ``Salt``
+| System set manager (SSM) | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Service pack migration | icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
+| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Traditional`` | icon:check[role="green"] | ``Salt``
+| Advanced Virtual Guest Management icon:asterisk[role="none"] | icon:times[role="danger"] | ``Salt`` | ``Salt``
+| Virtual Guest Installation (Kickstart), as Host OS | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
+| Virtual Guest Installation (image template), as Host OS | ``Traditional`` | icon:check[role="green"] | ``Salt``
+| System redeployment ({ay}) | ``Traditional`` | icon:check[role="green"] | ``Salt``
```

Comment:
Not sure if the macro {ay} is here correct. We for redhat it is "kickstart", not "autoyast"